telegram/ext/conversationhandler.py

```diff
@@ -423,6 +443,20 @@ def _get_key(self, update: Update) -> Tuple[int, ...]:
 
         return tuple(key)
 
+    def _resovle_promomise(self, state: Tuple) -> object:
```

Comment:
resovle promomise, hihi
```suggestion
    def _resolve_promise(self, state: Tuple) -> object:
```

Reply:
Oh shit!